runtime/Cpp/runtime/src/atn/LexerActionType.h

```diff
@@ -5,6 +5,8 @@
 
 #pragma once
 
+#include <stddef.h>
+
```

Comment:
I see, maybe we should follow the pattern here I used in other header files. They either include antlr4-common.h or a file which includes that.